client/src/main/scala/drt/client/components/DashboardTerminalSummary.scala

```diff
@@ -102,7 +101,7 @@ object DashboardTerminalSummary {
     }
   }
 
-  val aggSplits: Seq[ApiFlightWithSplits] => Map[PaxTypeAndQueue, Int] = BigSummaryBoxes.aggregateSplits(ArrivalHelper.bestPax)
+  def aggSplits: Seq[ApiFlightWithSplits] => Map[PaxTypeAndQueue, Int] = BigSummaryBoxes.aggregateSplits(ArrivalHelper.bestPax)
```

Comment:
Should this be a `val` still rather than a `def`? it only matters if it's being called more than a couple of times

Reply:
yes, probably! I'd added a parameter and then removed it, will make it a val again